public/locales/en/translation.json

```diff
@@ -34,5 +34,29 @@
   "psychotherapist": {
     "name": "Turgut Bamsi",
     "title": "Psychotherapist"
-  }
+  },
+  "weCare":{
+    "heading":"WE CARE AND CONCERN ABOUT YOU",
+    "firstword":"Lets",
+    "paragraph0":"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad",
+    "span0":"minim",
+    "paragraph1":"veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla",
+    "span1":"pariatur",
+    "paragraph2":".Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
+    "button0":"TALK TO US",
+    "button1":"If you are not sure, take the test"
+},
+"stories":{
+    "heading":"STORIES ABOUT HOPE",
+    "paragraph":"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. ",
+    "readmore":"read more >>",
```

Comment:
camelcase readMore. work with @gbilgedal to come up with better names for the translation keys perhaps (more informative ones), we generally try to avoid names like span0 paragraph0 etc